src/test/java/dev/blaauwendraad/masker/json/path/JsonPathTest.java

```diff
@@ -0,0 +1,24 @@
+package dev.blaauwendraad.masker.json.path;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+final class JsonPathTest {
+
+    @ParameterizedTest
+    @MethodSource("validJsonPaths")
+    void jsonPathParsing(String jsonPathLiteral) {
+        Assertions.assertEquals(jsonPathLiteral, JsonPath.from(jsonPathLiteral).toString());
+    }
+
+    private static Stream<String> validJsonPaths() {
+        return Stream.of(
+                "$.a.b",
+                "$.a",
+                "$..a.b."
```

Comment:
It is supposed to mean the path to "value" field in a "{"":{"a":{"b":{"":"value"}}}}" kind of json. You are right though that it might not follow the json path specification, so I will rework this part as well.

Reply:
Done.